### PR TITLE
Feature/find broken links

### PR DESCRIPTION
**Pull request by davidebukali**
1. Install wagtail link checker and celery plugins `pip3 install -r requirements.txt`

2. Run the migrations

3. Install Rabbitmq in Ubuntu `sudo apt-get install rabbitmq-server` for other [operating systems](http://docs.celeryproject.org/en/latest/getting-started/first-steps-with-celery.html#choosing-a-broker)

4. Add `export DJANGO_SETTINGS_MODULE=di_website.settings.dev` to your `activate` file inside `path_to_virtual_env_folder/bin/`

5. Run Celery worker with `celery -A wagtaillinkchecker worker -l info`

6. Open wagtail Settings Menu to access link checker menu and perform scan.

For details see plugin page - [here](https://github.com/neon-jungle/wagtail-linkchecker)

### CODE REVIEW

**Comment by edwinmp** [comment]
@davidebukali  Travis is breaking ... not review ready till it's happy.

**Comment by davidebukali** [comment]
> @davidebukali Travis is breaking ... not review ready till it's happy.

@edwinmp seems Travis was complaining about an app `wagtaillinkchecker` in  `INSTALLED_APPS` that had not been installed; removed it and tests now pass.

**Comment by davidebukali** [comment]
@edwinmp please install [wagtaillinkchecke](https://github.com/neon-jungle/wagtail-linkchecker); There's more explanation in my first comment. Including the plugin in requirements.txt breaks travis thats why I also removed it from `INSTALLED_APPS`

**Comment by edwinmp** [comment]
Hi @akmiller01 need a third opinion on this. Is it adequate?

**Comment by akmiller01** [comment]
Could you please add the relevant plugin to `requirements.txt` @davidebukali ? And then you can add it to `INSTALLED_APPS` to get this plugin working properly.

**Comment by davidebukali** [comment]
@akmiller01 sure thing

File: requirements.txt [hunk 1]
```diff
@@ -8,9 +8,27 @@ django-debug-toolbar==2.0
...
+pkg-resources==0.0.0
+psycopg2==2.8.2
+pylint==2.3.1
 python-decouple==3.1
 num2words==0.5.10
 beautifulsoup4==4.6.0
 bleach==3.1.0
-elasticsearch>=6.0.0,<6.3.1
\ No newline at end of file
+celery>=4.2.0rc4
```
Comment (edwinmp) [inline requirements.txt]: Instead of a release candidate, why not version 4.3.0? https://pypi.org/project/celery/

Comment (davidebukali) [reply requirements.txt]: Just tested with 4.3.0 and it works as well. It has been added and commited to the requirements.txt

**Pull request merged**